Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2008,13 +2015,34 @@ - (void)_acquireResizeAssertionForReason:(NSString *)reason
         return;
 
     if (_resizeAssertions.isEmpty()) {
-        [self _doAfterNextVisibleContentRectUpdate:makeBlockPtr([weakSelf = WeakObjCPtr<WKWebView>(self)] {
+        BOOL didInvalidateResizeAssertions = NO;
+
+        [self _doAfterNextVisibleContentRectUpdate:makeBlockPtr([weakSelf = WeakObjCPtr<WKWebView>(self), &didInvalidateResizeAssertions] {
```

Comment:
You're passing a reference to a stack-allocated bool into the blocks? This does not seem kosher.

Reply:
it's times like these that I miss JavaScript 🥲